### PR TITLE
Add copyright check

### PR DESCRIPTION
**Pull request by dj2**
This CL imports the copyright check from shaderc and adds it to the
check-format kokoro bot.

### CODE REVIEW

File: tools/copyright.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# Copyright 2018 The Amber Authors.
```
Comment (dj2) [inline tools/copyright.py]: Switched this from Shaderc to Amber, given we have the copyright on both I figured that was OK?

Comment (dneto0) [reply tools/copyright.py]: Yes, all contributions to that script in the Shaderc repo were from Googlers, so Google owns the copyright to the original contributions.

File: tools/copyright.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    break
+        if not has_copyright:
+            print(file, ' has no copyright message.')
+            printed_count += 1
+    return printed_count
+
+
+def main():
+    glob_comment_pairs = [('*.h', '//'), ('*.hpp', '//'), ('*.cc', '//'),
+                          ('*.py', '#'), ('*.cpp', '//')]
```
Comment (dneto0) [inline tools/copyright.py]: I'd add .mm to this list.  :-)

Comment (dj2) [reply tools/copyright.py]: done

**Comment by dneto0** [comment]
The check-format failure is because there was no code to format.  Spurious failure.

**Pull request merged**